strawberry_django/fields/types.py

```diff
@@ -1,10 +1,11 @@
 import datetime
 import decimal
 import uuid
-from typing import List, Optional
+from typing import List, Optional, Tuple, Union
 
 import django
 import strawberry
+from django.contrib.gis.db import models as geos_fields
```

Comment:
Please remove this import. geos_fields is reimported in try block below. pre-commit check fails due that